crud/bid.py

```diff
@@ -47,5 +47,18 @@ def update_estimates(
     return bid_estimates
 
 
-def get(db: Session):
-    return db.scalars(select(Bid)).all()
+def get(
+    db: Session,
+    approved: bool | None = None,
+    bid_manager_id: int | None = None,
+    customer_id: int | None = None,
+):
+    query = select(Bid)
+    if approved is not None:
```

Comment:
You should be able to replace this with ```if approved: ```

Reply:
can't because approved can be either boolean or None. I only want to add the filter if it's not None